### PR TITLE
Allow custom NTP servers

### PR DESCRIPTION
**Pull request by e-minguez**

### CODE REVIEW

**Comment by russellb** [comment]
Can we please retain "make this work automatically within red hat"?

The extra customization ability is nice, though.  Thanks.

File: utils.sh [hunk 1]
```diff
@@ -26,10 +26,15 @@ function create_cluster() {
     $GOPATH/src/github.com/openshift-metalkube/kni-installer/bin/kni-install --dir "${assets_dir}" --log-level=debug create manifests
 
     # TODO - consider adding NTP server config to install-config.yaml instead
-    if host clock.redhat.com ; then
-        cp assets/templates/99_worker-chronyd-redhat.yaml.optional assets/templates/99_worker-chronyd-redhat.yaml
-        cp assets/templates/99_master-chronyd-redhat.yaml.optional assets/templates/99_master-chronyd-redhat.yaml
+    host clock.redhat.com > /dev/null && NTP_SERVERS="clock.redhat.com"
```
Comment (karmab) [inline utils.sh]: this means that if run within redhat network, you ll get this specific ntp server regarless of the option set for NT_SERVERS

Comment (e-minguez) [reply utils.sh]: Fixed I believe

**Comment by russellb** [comment]
@e-minguez I'm good with adding the patch here, but it'll have to be rebased.  Sorry about that

**Comment by e-minguez** [comment]
> @e-minguez I'm good with adding the patch here, but it'll have to be rebased. Sorry about that

Rebased!

File: extras/pre-00.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```
Comment (russellb) [inline extras/pre-00.sh]: I think we dropped the extras/ directory, so I'd say we can drop this file

Comment (e-minguez) [reply extras/pre-00.sh]: Dropped

**Comment by russellb** [comment]
thanks - just waiting for a CI run now

**Comment by e-minguez** [comment]
> thanks - just waiting for a CI run now

:crossed_fingers:

**Comment by derekhiggins** [comment]
Build SUCCESS, see build http://10.8.144.11:8080/job/dev-tools/661/

**Pull request merged**